### ISSUE TITLE
[Progress-bar] Component specs

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#227]
@GauravMaurya0

I've just created the brach you should work against:
https://github.com/dxc-technology/halstack-style-guide/tree/component-progress/guidelines/components/progressbar

- [x] Remove **modes** section
- [x] Remove **states** section
- [x] Add section named: **Determinate or indeterminate** with the following content:
```
## Determinate or indeterminate   
 
![Determinate progress bar](images/progress_determinate.png)

_Determinate progress bar_

* Determinate indicators display how long a process will take. They should be used in longer processes.

![Indeterminate progress bar](images/progress_indeterminate.png)

_Indeterminate progress bar_

* Indeterminate indicators express an unspecified amount of wait time. They should be used when:
        * The processing time is unknown.
        * The wait time is expected to be short enough that it’s not necessary to display.          
```
- [x] Move **design tokens** table to  **design specifications** section named **color** subsection
- [x] Update all specs tables following [content contribution guidelines](https://github.com/dxc-technology/halstack-style-guide/blob/3.3.0/contributing/content.md)
- [x] Update all documentation images according to the [image contribution guidelines](https://github.com/dxc-technology/halstack-style-guide/blob/3.3.0/contributing/images.md)
- [x] Add bottom links  to the component page

**Comment by aweell** [comment]
Component `progress-bar` has been added to the UI-Kit

**Comment by aweell** [comment]
Closed via #314 and #315
